src/org/traccar/database/DeviceManager.java

```diff
@@ -317,80 +317,43 @@ public void removeGroup(long groupId) throws SQLException {
         groupsById.remove(groupId);
     }
 
-    public boolean lookupServerBoolean(long deviceId, String attributeName, boolean defaultValue) {
-        String result = lookupAttribute(deviceId, attributeName, true);
+    public boolean lookupAttributeBoolean(long deviceId, String attributeName, boolean defaultValue,
+            boolean lookupServer) {
+        String result = lookupAttribute(deviceId, attributeName, lookupServer);
         if (result != null) {
             return Boolean.parseBoolean(result);
         }
         return defaultValue;
     }
 
-    public String lookupServerString(long deviceId, String attributeName, String defaultValue) {
-        String result = lookupAttribute(deviceId, attributeName, true);
+    public String lookupAttributeString(long deviceId, String attributeName, String defaultValue,
```

Comment:
Same here, it's better to move all parameters on the next line if it doesn't fit.